server/fishtest/templates/base.mak

```diff
@@ -150,11 +150,36 @@
                     title="Upload Neural Network"
                   >
                     <i
-                      class="fa-solid fa-cloud-arrow-up d-inline me-2 mx-lg-1"
+                      class="fa-solid fa-cloud-arrow-up d-inline me-2 px-lg-1"
```

Comment:
My fault for not being more clear. I meant changing `nav-link py-2 px-0 px-lg-2` to `nav-link py-2 px-0 px-lg-1` a bit above this.

Reply:
I changed this. But should the change on line 153 you quote be undone? 

(I have no idea what these bootstrap incantations mean).